GiEnJul/Controllers/AdminController.cs

```diff
@@ -28,13 +28,13 @@ public AdminController(IEventRepository eventRepository, IGiverRepository giverR
             _mapper = mapper;
         }
 
-        [HttpGet("querygivers")]
+        [HttpGet("recipients/filter=unmatched")]
```

Comment:
I mistyped earlier.. This should have been `recipients?filter=unmatched`

Reply:
Yep, It seems a bit more difficult to implement routing when you have inputs and we should ask Ronny about best practices. I'm going to create an issue for it, and comment out this function in this PR.